### ISSUE TITLE
Loss of Yield as High

### ISSUE THREAD

**Issue by GalloDaSballo** [code-423n4/org#96]
Am flagging this Judging Decision as a way to avoid it being used again by other wardens in the future
https://github.com/code-423n4/2023-02-ethos-findings/issues/482

The finding was judged as High with the following comment:
> [Personally view loss of yield as high severity (in line with Immunefi classification).](https://github.com/code-423n4/2023-02-ethos-findings/issues/482#issuecomment-1476068586)

This is in my opinion an incorrect explanation because the contest should be judged via the rules from CodeArena, those are the rules all Wardens have opt-ed into, external rules and classification should not be a valid rationale for determining severity.

By the same logic all Consensys findings flagged as Medium would have been accepted when Medium Severities are not even exploits by their own classification

There are hundreds of reports that have classified as Med via the "leak of value" interpretation of the rules, in my opinion this should have been judged in the same way.

Interested in more Judges thoughts

@trust1995 @dmvt @0xean  @0xean @0xleastwood

**Comment by trust1995** [comment]
To be clear I didn't regard it as high because Immunefi defines it as such, it's only a supporting statement. 
Loss of yield == loss of principal for me as both are monetary loss for users. If there are specific scenarios where this can occur, it's a med, if it's pretty much directly trigger able than high. If requires explicit user mistake, than it's probably a QA leak of value.

**Comment by dmvt** [comment]
I probably would have judged this medium because it's a value leak more than a direct exploit and the only in certain circumstances. That said, I can see this as a high risk as well and ultimately it's up to @trust1995 as the judge.

I do agree fully with @GalloDaSballo that we should refrain from referencing other bounty / audit systems in our judgements. It adds no value. A judge's reasoning should stand on its own with any rule or standard followed being from C4 docs / issues / discussions.

**Comment by gjaldon** [comment]
According to the [code4rena documentation](https://docs.code4rena.com/awarding/judging-criteria/severity-categorization), it says:

> 3 — High: Assets can be stolen/lost/compromised directly (or indirectly if there is a valid attack path that does not have hand-wavy hypotheticals).

Rewards are in the form of tokens and they are assets. Did I misunderstand? 

This looks like a High severity to me because there is a direct loss of assets. Not only do the tokens not get distributed to the intended recipients, they will remain stuck in the contract forever.  The contract is not upgradable and it has no centralized functionality for withdrawing tokens.

**Comment by 0xleastwood** [comment]
> According to the [code4rena documentation](https://docs.code4rena.com/awarding/judging-criteria/severity-categorization), it says:
> 
> > 3 — High: Assets can be stolen/lost/compromised directly (or indirectly if there is a valid attack path that does not have hand-wavy hypotheticals).
> 
> Rewards are in the form of tokens and they are assets. Did I misunderstand?
> 
> This looks like a High severity to me because there is a direct loss of assets. Not only do the tokens not get distributed to the intended recipients, they will remain stuck in the contract forever. The contract is not upgradable and it has no centralized functionality for withdrawing tokens.

I would disagree, rewards are yield that is paid on top of the principal investment. User's assets are still safe and remain unaffected, however, I think it also depends to what extent rewards can be stolen. Rounding errors does not impact the protocol too much but if a malicious user can arbitrarily claim other users' rewards, I'd say that is pretty severe and worthy of being `high` severity. Again, context and impact matters.

**Comment by aktech297** [comment]
> To be clear I didn't regard it as high because Immunefi defines it as such, it's only a supporting statement. Loss of yield == loss of principal for me as both are monetary loss for users. If there are specific scenarios where this can occur, it's a med, if it's pretty much directly trigger able than high. If requires explicit user mistake, than it's probably a QA leak of value.

how would be the severity in the case of staking type of scenario. For example, chain-link project.?

the ultimate aim of staking is to get reward for the work which is done by the user.

If reward is lost due to mistake in the code , this render the staking meaningless. I would see this High.  Looking for further feedback.

Thanks!

**Comment by trust1995** [comment]
Best to discuss the standardization of the topic in the SC discussion. We have definitions for when this is high and med.

**Comment by aktech297** [comment]
There is an issue in Chain link contest which is related to this. 
https://github.com/code-423n4/2023-08-chainlink-findings/issues/832

Sure, I will raise this to supreme court. But I am worried that the time is very less . So, come back here.

Thanks for your kind response
